_gsdocs-proposals/2019/proposal_DIRAC.md

```diff
@@ -0,0 +1,39 @@
+---
+title: DIRAC - the interware
+layout: gsdocs_proposal
+project: DIRAC
+year: 2019
+organization:
+ - CERN
+---
+
+## Description
+
+[DIRAC](http://diracgrid.org/) is a complex, open source, very actively developed software. The DIRAC project is a complete Grid solution for one, or more than one community of users that need to exploit distributed, heterogeneous resources. DIRAC forms a layer between a community and various compute resources to allow optimized, transparent and reliable usage. DIRAC's roles range from the submission of jobs, the management of the data produced, to the orchestration of the distributed resources, while providing active monitoring to its users. DIRAC is a generic software, used and extended by several Virtual Organizations (VO). One of the communities that use DIRAC is [LHCb](http://lhcb-public.web.cern.ch/lhcb-public/), the LHC "beauty" experiment, which is DIRAC's initiator and main contributor.
+
+The writer will contribute in improving the DIRAC documentation and tutorials, and to eliminate duplicate or outdated parts from the current one. The is available from multiple places and in different formats, including also those found in [scientific articles](https://cds.cern.ch/record/2631792). The writer will concentrate on improving the documentation that can be found on [readthedocs.io](https://dirac.readthedocs.io/en/latest/), which has source in the [code](https://github.com/DIRACGrid/DIRAC/tree/integration/docs). The writer will also look for documentation written in the [Wiki](https://github.com/DIRACGrid/DIRAC/wiki), and possibly merge into the one published in readthedocs.io.
+
+
+
+## Tasks
+
+Possible tasks might include:
+
+- Test the existing tutorials and users' documentation, e.g. by trying to install DIRAC.
+- Review the existing documentation, and propose guidelines for improving it. Discuss the proposal with the authors.
+- Find duplicated and/or possibly outdated parts of the documentation.
+- Re-arrange the documentation architecture according to best practices.
+- Improve the automatically built code and configuration documentation.
+
+
+## Mentors
```

Comment:
Please insert the mentors to the _activities/gsoc.md file, ordered by last name.